Alan/coo 1002 update docs customise consent UI (#28)
IdentitySquare/CookieChimp-Docs: docs/features/banner-designer.mdx

---
title: "Banner Designer"

> description: "Customize the consent banner & preferences modal to match your branding."

Reviewer: ```suggestion
description: "Customise the consent banner & preferences modal to match your branding."
```

Reviewer: we want to use EU spelling since most of our clients are EU and there are more regulations here anyway.

---

<img src="/images/consent-banner/banner-designer.png" alt="Banner Designer" />

## Geo Targeting

> Banners can be set up to show worldwide or only in certain countries or US states.
> 
> Multiple banners can be set up in this way to match the user's country and their regulations.

Reviewer: ```suggestion
Banners can be configured to show only in specific countries or U.S. states. This lets you create multiple banners customised for different regions and their regulations.
```

<img src="/images/consent-banner/geo-targeting.png" alt="Geo Targeting" />

## Force Consent

The force consent option can be toggled to increase consent rates.

When this option is enabled, the user is forced to accept/deny consent before interacting with the website.

<img src="/images/consent-banner/force-consent.png" alt="Force Consent" />

## Layouts

> The layout, position and button placement of banners can be customized to fit your website's style.

Reviewer: ```suggestion
The layout, position and button placement of banners can be customised to fit your website's style.
```

<img src="/images/consent-banner/banner-layout.png" alt="Banner Layout" />

> Similarly, the layout of the preferences modal can also be customized.

Reviewer: ```suggestion
Similarly, the layout of the preferences modal can also be customised.
```

<img src="/images/consent-banner/preferences-layout.png" alt="Preferences Layout" />

## Banner Buttons

> The weights, position and text of banner buttons can be customized.

Reviewer: ```suggestion
The weights, position and text of the banner buttons can be customised.
```

<Note>
  Some regulations like GDPR require banner buttons to be equally weighted.

>   Manage preferences button must be enabled to give fine granular control to the users.

Reviewer: ```suggestion
```

  Some regulations might require specific text for buttons.
</Note>

<img src="/images/consent-banner/banner-buttons.png" alt="Banner Buttons" />

## Privacy Trigger

> The privacy trigger that allows users to update their consent preferences at any time, can be enabled/disabled and its position can be customized.

Reviewer: ```suggestion
The privacy trigger is a floating button that allows users to update their consent preferences at any time. This can be enabled/disabled and its position can be customised.
```

> <Note>
>   Some regulations will require this feature to be enabled.
> </Note>

Reviewer: ```suggestion
<Note>
  Most regulations mandate that users must be able to update their consent preferences after giving their initial consent. You can enable this using our Privacy Trigger or by adding your own button or link to open the preferences.
</Note>
```

link the you can add your own button to the other page where we talk about the syntax to open the modal with the data attribute

<img src="/images/consent-banner/privacy-trigger.png" alt="Privacy Trigger" />

## Preferences Control Level

> The preferences modal can be customized to only show categories or to show their services and cookies and allow users to have more granular control.

Reviewer: ```suggestion
The preferences modal can be customised to only show categories or to show their services and cookies and allow users to have more granular control.
```

<Note>

> Some regulations will require all storage items and services to be shown.

Reviewer: ```suggestion
Some regulations require all storage items and services to be shown.
```

</Note>

<img src="/images/consent-banner/control-level.png" alt="Control Level" />

## Theme

> The banner's appearance can be customized with various theme options:

Reviewer: ```suggestion
The banner's appearance can be customised with various theme options:
```

- Dark/Light mode - Choose between light, dark, or auto mode that follows the user's system preferences

> - Colors - Customize the banner's primary colors, text colors, and button colors to match your brand

Reviewer: ```suggestion
- Colors - Customise the banner's primary colors, text colors, and button colors to match your brand
```
